geomesa-fs/geomesa-fs-tools/bin/install-hadoop.sh

```diff
@@ -18,7 +18,7 @@ htrace_core_version="4.1.0-incubating"
 
 # for hadoop 2.5 and 2.6 to work we need these
```

Comment:
Do the comments in the install-hadoop.sh scripts need to be updated?  (Hate to ask, but this may be a good time to grep for 2.5/2.6.)

Reply:
i think the comments are still correct? i'm not 100% sure though so i didn't want to just remove them.

Reply:
I just wonder if we want to drop any mention of unsupported versions of Hadoop?

Reply:
ah, i forgot we had dropped support for hadoop before 2.8. let me take a look...

Reply:
hadoop 2.8 still uses the same version, so i just updated the comment